srcpkgs/qbe/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'qbe'
+pkgname=qbe
+version=1.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+short_desc="Compiler backend"
```

Comment:
This *is* the only short description in the webpage, but I feel like it's too short.... Not sure how to expand it though. Are there any other distros with a QBE package already? Maybe we could take inspiration from them.

Reply:
"A simple C compiler backend" is the description of the package on Alpine Linux.

Reply:
Maybe "Small-footprint compiler backend with full support for the C ABI" or something along those lines?